### ISSUE TITLE
Represent line breaks in tweets

### ISSUE THREAD

**Issue by hyperlink** [cezarsa/silver_bird#67]

**Comment by cezarsa** [comment]
Actually tweets don't support line breaks, I think the best thing to do would be stop supporting line breaks in the textarea.

**Comment by hyperlink** [comment]
I noticed Twitter.com doesn't display them but people do use them and it seems perfectly natural when displayed in clients like Tweetie.

Also If line breaks should not allowed then wouldn't enter key be able to send a tweet? Yet CTRL-ENTER is used instead which to me implies line breaks are a possibility. :)

**Comment by cezarsa** [comment]
I wasn't aware that other clients allowed line breaks, so I think it's okay if we support them too. :)
I just received an e-mail with a patch from Xiaoxin Lu adding this feature. I'm going to apply it as soon as I get home.

**Comment by cezarsa** [comment]
Oh, I just noticed that you're the guy who sent the patch. :)

I just pushed it, thank you very much for your contribution, I'd love to see lots of people contributing just like you did because that's what drives open source projects forward.

Also answering your e-mail, yes, git format-patch is the best way to contribute (or making a pull request from a cloned repo) because it lets you specify the commit message and also lets you take credit for the commit. Thank you again.

Closed by f6ec22bc3c0b48cfd757d7a3b8fcc9dbc7d1750c
